doc/whatsnew.rst

```diff
@@ -1,15 +1,18 @@
 What's new
 **********
 
-Next release
-============
+.. Next release
+.. ============
+
+v2024.8.6
+=========
 
 - Add :doc:`/material/index` (:pull:`188`, :pull:`189`).
 - Update :doc:`/material/index` (:pull:`201`).
-- Add :doc:`/transport/index` (:pull:`207`, :pull:`208`).
+- Add :doc:`/transport/index` (:pull:`207`, :pull:`208`, :issue:`210`, :pull:`212`).
 - Add :doc:`/project/edits` project code and documentation (:pull:`204`).
 - Reduce log verbosity of :func:`.apply_spec` (:pull:`202`).
-- Made fixes and updates to :doc:`/api/tools-costs` (:pull:`186`, :pull:`187`, :pull:`190`, :pull:`195`).
+- Fixes and updates to :doc:`/api/tools-costs` (:pull:`186`, :pull:`187`, :pull:`190`, :pull:`195`).
```

Comment:
Are we just using verbs (as above)? Then maybe this should be
```suggestion
- Fix and update :doc:`/api/tools-costs` (:pull:`186`, :pull:`187`, :pull:`190`, :pull:`195`).
```

Reply:
Yes, that's better, thanks! Imperative mood, present tense.